src/SoftDeleteable/Query/TreeWalker/Exec/MultiTableDeleteExecutor.php

```diff
@@ -20,21 +20,19 @@ class MultiTableDeleteExecutor extends BaseMultiTableDeleteExecutor
     /**
      * {@inheritdoc}
      */
-    public function __construct(Node $AST, $sqlWalker, ClassMetadataInfo $meta, AbstractPlatform $platform, array $config)
+    public function __construct(Node $AST, $sqlWalker, ClassMetadata $meta, AbstractPlatform $platform, array $config)
```

Comment:
Technically could be a B/C break.  But, in practice, Doctrine isn't passing through the parent `ClassMetadataInfo` class, it typehints to `ClassMetadata` somewhat heavily, and this class really isn't meant for outside direct use.

Reply:
Could we mention this in the changelog?

Reply:
Done